### ISSUE TITLE
No rule to make target 'vg_guest_client'

### ISSUE THREAD

**Issue by Druidpeter** [maquefel/virtual_gpio_basic#3]
When I run the compilation command: `KDIR=/usr/src/linux-5.15.19/ QEMU_DIR=/home/<homedir>/Clones/qemu-7.1.0/ make all
`

the process fails with "No rule to make target 'vg_guest_client', needed by 'all'. Checking the Makefile shows that there is indeed no rule inside the file, and I'm not sure how create the rule, since I don't know what vg_guest_client consists of.

How can I compile this?

**Comment by maquefel** [comment]
Yep - vg_guest_client dropped a while ago without fixing targets.

Please follow https://github.com/maquefel/kernel-bisect-template/blob/virtual_gpio/Makefile#L97 to compile vg_get_set and virtual_gpio module.

Or use https://github.com/maquefel/kernel-bisect-template/tree/virtual_gpio.
